src/beast/extras/beast/unit_test/suite_info.hpp

```diff
@@ -87,9 +90,12 @@ class suite_info
     bool
     operator<(suite_info const& lhs, suite_info const& rhs)
     {
-        return
-            std::tie(lhs.library_, lhs.module_, lhs.name_) <
-            std::tie(rhs.library_, rhs.module_, rhs.name_);
+        if (lhs.priority_ == rhs.priority_)
+            return std::tie(lhs.library_, lhs.module_, lhs.name_) <
+                   std::tie(rhs.library_, rhs.module_, rhs.name_);
+        else
+            // we want higher priority suites sorted first
+            return lhs.priority_ > rhs.priority_;
```

Comment:
Could just add `priority_` to the tuple and negate it.
```
return std::tie(-lhs.priority_, lhs.library_,...) < std::tie(-rhs.priority_, rhs.library_,...)
```

Reply:
I like the suggestion, but I think I need to switch to `make_tuple` in that case (?) b/c `tie` expects lvalues.

Reply:
I don't think I'd use `make_tuple` because it's going to be too easy to make a copy of member variables. We could assign the negative value to a local, but I'm not sure that's any better than what's there now. I'd probably keep what's there.

Reply:
Actually, I think `std::forward_as_tuple` would work.